Code/GraphMol/AddHs.cpp

```diff
@@ -73,7 +73,7 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
       // --------------------------------------------------------------------------
       dirVect.z = 1;
       // loop over the conformations and set the coordinates
-      for (ROMol::ConformerIterator cfi = mol->beginConformers();
+      for (auto cfi = mol->beginConformers();
```

Comment:
I was formulating a similar wish (also for atoms and bonds) this morning. Glad to see that there's likely a simple-ish solution.